app/flows/covid_travel_abroad_flow/outcomes/_fully_vaxed.erb

```diff
@@ -20,13 +20,10 @@
   <p class="govuk-body">You must:</p>
 
   <ul class="govuk-list govuk-list--bullet">
-    <li><a href="https://www.gov.uk/find-travel-test-provider" class="govuk-link">take a PCR or rapid lateral flow COVID-19 test</a> - to be taken in the 48 hours before you travel to England</li>
-    <li><a href="https://www.gov.uk/find-travel-test-provider" class="govuk-link">book and pay for a COVID-19 PCR test</a> - to be taken after arrival before the end of day 2</li>
+    <li><a href="https://www.gov.uk/find-travel-test-provider" class="govuk-link">book and pay for a COVID-19 PCR or rapid lateral flow test</a> - to be taken after arrival before the end of day 2</li>
```

Comment:
The doc has 'you must take it after arrival before the end of day 2' rather than 'to be taken after arrival before the end of day 2'. I checked with Jimmy but haven't heard back, its not highlighted as a change on the doc so not 100% sure.

Reply:
Jimmy confirmed that it needs to change to 'you must take it after arrival before the end of day 2'

Reply:
Sorry I missed this - it wasn't highlighted as a change on doc 😞 Thanks for spotting it.  Done.